MyFile.txt

```diff
@@ -1,2 +1,4 @@
 Hi, this is my first file I am creating through branching
+I have added this second line
```

Comment:
Hi, i know this is the second line, is it necessary? I think you could delete that